packages/www/components/Dashboard/WebhooksTable/index.tsx

```diff
@@ -118,6 +126,28 @@ const WebhooksTable = ({ title = "Webhooks" }: { title?: string }) => {
                 cursor: "pointer",
               },
             },
+            status: {
+              children: (
+                <Box>
+                  {!webhook.status ? (
+                    "-"
```

Comment:
I think the `Idle` status badge would be not the correct one here (when the status is missing), in the stream `-` is displayed, but probably would be better an empty string?

Reply:
I think `Idle` makes sense tho, as the meaning of a webhook that hasn't received any events. Maybe the tooltip could even explain it like `"This webhook hasn't received any events yet"` (gotta update the `tooltipText` logic to allow one with no timestamps tho).

If you think there would be some other more accurate term, you can also create new `StatusVariant`s for that. I'm also ok with `-` anyway if that looks ok in the table.